vivisect/base.py

```diff
@@ -786,11 +786,13 @@ def _cb_opcode(self, va, op, branches):
 
             self._mem.makeOpcode(op.va, op=op)
             # TODO: future home of makeOpcode branch/xref analysis
-            return branches
+            if not self._mem.isNoReturnVa(op.va):
+                return branches
```

Comment:
I'm not happy this is here, but until we migrate codeflow to being full emulation and not the flyweight emulation it is, this is necessary to stop the branching early in certain syscall cases.

Reply:
It's specifically because I'm having to stop the emulator and mark things as no return in the analysis monitor that I have to have this here to tell codeflow to not carry on.

Reply:
ahhh, because getBranches doesn't check to see if a branch is noret?

Reply:
Yup.